### ISSUE TITLE
PredictiveMind: default belief-update thresholds make predictive inference effectively inert

### ISSUE THREAD

**Issue by Tristan-Stoltz-ERC** [Luminous-Dynamics/symthaea#2005]
## Finding

The production `PredictiveMind` used by `CognitiveLoopService` contains a serious update-law mismatch in `src/consciousness/dynamics/predictive_processing.rs`.

`PredictiveLayer` defaults to `learning_rate = 0.1`. Its two belief-update paths compute:

```text
bottom-up blend_factor = weighted_error × recognition_confidence × learning_rate
top-down  blend_factor = prediction_precision × learning_rate
```

Under the model's documented/bounded quantities:

```text
weighted_error ≤ 1
recognition_confidence ≤ 1
prediction_precision ≤ 1
learning_rate = 0.1
```

so both default blend factors are bounded by `0.1`.

However `blend_beliefs()` leaves the current belief unchanged for every factor `≤ 0.3`, binds only for `> 0.3`, and replaces with the incoming belief only for `> 0.5`.

Therefore, under default configuration, both the bottom-up prediction-error path and top-down prediction path appear unable to alter `PredictiveLayer::belief` at all. The system can still compute/store prediction errors, free energy, precision dynamics and telemetry, but the central predictive-inference belief update is effectively unreachable.

This matters directly to Butlin **PP-1** and **HOT-1** evidence and indirectly to any claim that `PredictiveMind` performs hierarchical predictive coding rather than prediction-error measurement only.

## Required correction

Replace the discontinuous/unreachable update rule with an update whose effective range is compatible with the configured learning rate and bounded precision/error terms.

Acceptable approaches include:

- a true continuous interpolation/update in an appropriate representation space;
- a probabilistic HDC bit-update whose expected update magnitude scales continuously with the blend factor;
- a threshold schedule explicitly derived from the configured learning-rate range, with tests proving reachability;
- another mathematically justified HDC update rule.

Do **not** fix this merely by raising the default learning rate above the current 0.3 threshold without stability evidence.

## Required tests

1. **Bottom-up update reachability** — after a learned recognition association, a nonzero precision-weighted prediction error must measurably change belief under default config.
2. **Top-down update reachability** — a high-precision prediction from above must measurably change the lower-level belief under default config.
3. **Dose response** — larger precision/error should produce a larger expected representational update than smaller precision/error.
4. **Zero control** — zero precision or zero effective error must produce no update.
5. **Bounded/stable update** — repeated updates remain finite/stable and do not collapse representation indiscriminately.
6. **Predictive consequence** — learning must reduce held-out prediction error for a repeated/structured relation compared with a frozen-learning control.

## Butlin evidence consequences

Until this is fixed and directly qualified:

- `actual_effective_lr > 0` must not be treated as evidence that predictive beliefs actually updated;
- PP-1 should require a prediction-error → model-update → reduced future error theorem;
- HOT-1 should require top-down generative state to causally alter perceptual inference under ambiguity/noise;
- the same fix/test evidence must not be counted twice as independent evidence for PP-1 and HOT-1; shared evidence dependencies should be explicit.

## Exit condition

Default `PredictiveMind` demonstrates reachable, graded bottom-up and top-down belief revision plus held-out prediction-error reduction under provenance-bound tests.

No consciousness claim is implied.
